doc/installation.md

```diff
@@ -1,104 +1,83 @@
-# Installation
+# Local Installation
+This guide will walk you through the recommended path to get Pachyderm running locally on OSX or Linux.
 
-## Common Prerequisites
+If you hit any errors not covered in this guide submit an issue on [GitHub](github.com/pachyderm/pachyderm) or email us at [support@pachyderm.io](mailto:support@pachyderm.io) and we can help you right away.  
 
-- [FUSE (optional)](#fuse-optional) >= 2.8.2
-- [Kubectl (kubernetes CLI)](#kubectl) >= 1.2.2
-- [pachctl Command Line Interface](#pachctl)
-- [Pachyderm Repository](#pachyderm)
+## Prerequisites
+- [Minikube](#minikube) (and VirtualBox)
+- [Pachyderm Command Line Interface](#pachctl)
 
-## FUSE (optional)
+### Minikube
 
-Having FUSE installed allows you to mount PFS locally, which can be nice if you want to play around with PFS.
+Kubernetes offers a fantastic guide to [install minikube](http://kubernetes.io/docs/getting-started-guides/minikube). Follow the Kubernetes installation guide to install Virtual Box, Minikibe, and Kubectl. Then come back here to install Pachyderm. 
 
-FUSE comes pre-installed on most Linux distributions.  For OS X, install [OS X FUSE](https://osxfuse.github.io/)
-
----
-
-## Kubectl
-
-Make sure you have version 1.2.2 or higher.
-
-```shell
-### Darwin (OS X)
-$ wget https://storage.googleapis.com/kubernetes-release/release/v1.2.2/bin/darwin/amd64/kubectl
-
-### Linux
-$ wget https://storage.googleapis.com/kubernetes-release/release/v1.2.2/bin/linux/amd64/kubectl
-
-### Copy kubectl to your path
-chmod +x kubectl
-mv kubectl /usr/local/bin/
-```
-
----
-
-## pachctl
+### Pachctl
 
 `pachctl` is a command-line utility used for interacting with a Pachyderm cluster.
 
-### Installation
-
-#### Homebrew
 
 ```shell
+# For OSX:
 $ brew tap pachyderm/tap && brew install pachctl
-```
-
-#### Deb Package
 
-If you're on linux 64 bit amd, you can use our pre-built deb package like so:
-
-```shell
+# For Linux (64 bit):
 $ curl -o /tmp/pachctl.deb -L https://pachyderm.io/pachctl.deb && dpkg -i /tmp/pachctl.deb
 ```
 
-#### From Source
+You can try running `pachctl version` to check that this worked correctly, but Pachyderm itself isn't deployed yet so you won't get a `pachd` version. 
 
-To install pachctl from source, we assume you'll be compiling from within $GOPATH. So to install pachctl do:
+```sh
+$ pachctl version
+COMPONENT           VERSION
+pachctl             1.1.0
+pachd               (version unknown) : error connecting to pachd server at address (0.0.0.0:30650): context deadline exceeded
 
-```shell
-$ go get github.com/pachyderm/pachyderm
-$ cd $GOPATH/src/github.com/pachyderm/pachyderm
-$ make install
+please make sure pachd is up (`kubectl get all`) and portforwarding is enabled
```

Comment:
This sentence seems really weird to me, we know that pachd isn't up yet so they won't find it when they do `kubectl get all` why are you having them do that?